wombat-admin/src/app/page.tsx

```diff
@@ -78,32 +78,51 @@ export default function Home() {
   const closeViewImageModal = (): void => setSelectedRequest(null)
 
 
+
+  // const approveRequest = async (requestId: string) => {
+  //   const request = manualOverridenRequests.find((req) => req.requestId === requestId);
+
+  //   const reqsBody = { state: 'MANUAL_OVERRIDE_ACTIONED', date: request!.date }
+
+  //   await fetch(`${URL}/${accessPath}/${requestId}`, {
+  //     method: "PUT",
+  //     headers: {
+  //       "Content-type": "application/json"
+  //     },
+  //     body: JSON.stringify(reqsBody)
+  //   })
+  //     .then((response) => response.json())
+  //     .then((data) => console.log(data))
+  //     .catch((error) => console.error(error));
+  // }
+
   const approveRequest = async (requestId: string) => {
     const request = manualOverridenRequests.find((req) => req.requestId === requestId);
 
     if (!request) {
-      console.error('Request not found');
+      console.error(`Request with ID ${requestId} not found.`);
       return;
     }
 
-    const requestOptions = {
-      method: 'PUT',
-      headers: {
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify({ state: 'MANUAL_OVERRIDE_ACTIONED', date: "2024-04-17-05:00" }),
-    };
+    const reqsBody = { state: 'MANUAL_OVERRIDE_ACTIONED', approvalStatus: "DENIED", date: request.date };
```

Comment:
if this is the approve request hook it should have an approvalStatus of "APPROVED"